lib/Red/AST/Optimizer/AND.pm6

```diff
@@ -1,12 +1,14 @@
 use Red::AST;
 use Red::AST::Infixes;
 use Red::AST::Value;
+
 unit role Red::AST::Optimizer::AND;
 
 #| x > 1 AND x > 10 ==> x > 10
-multi method optimization-col1(
+multi method optimize(
     $left  where Red::AST::Ge|Red::AST::Gt,
-    $right where Red::AST::Ge|Red::AST::Gt
+    $right where Red::AST::Ge|Red::AST::Gt,
```

Comment:
It looks like you've typed some of these often enough to warrant a  subset or two :-)